starlite/contrib/htmx/request.py

```diff
@@ -0,0 +1,103 @@
+import json
+from functools import cached_property
+from typing import Any
+from urllib.parse import unquote, urlsplit, urlunsplit
+
+from starlite import Request
+
+
+class HtmxDetails:
+    """HtmxDetails holds all the values sent by HTMX client in headers
+    and provide convenient properties.
+    """
+
+    def __init__(self, request: Request) -> None:
+        """Initialize Class"""
+        self.request = request
+
+    def _get_header_value(self, name: str) -> Any:

```

Comment:
All of the methods and properties on this class are typed to return `Any`, is that for a particular reason?

Reply:
I am currently working on detail documentation and README.